### PR TITLE
echam distrub recipe

### PR DESCRIPTION
**Pull request by pgierz**
How to apply an echam disturbance.

Probably, it would be nice to have the exp tree from a graphviz, but I didn't know how to do that.

### CODE REVIEW

File: docs/recipes/echam_disturbance.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+      * EXPID
+      |
+      | * scripts
+      | | - run_config.yaml
+      | | - distrub_years.dat
+      | |
+      | * outdata
+      | | - echam
+      | ... other files and folders ...
```
Comment (mandresm) [inline docs/recipes/echam_disturbance.rst]: Do you mean to turn this into a tree in Graphviz? I can do that if you want.

File: docs/cookbook.rst [hunk 2]
```diff
@@ -35,6 +35,7 @@ documentation issue on `our GitHub repository <https://github.com/esm-tools/esm_
 .. } (Move to the end of the paragraph)
 .. : sort (sort the lines you have selected alphabetically)
 
-.. include:: recipes/modify_namelists.rst
 .. include:: recipes/add_model_setup.rst
+.. include:: recipes/echam_disturbance.rst
+.. include:: recipes/modify_namelists.rst
 .. include:: recipes/use_own_namelist.rst
```
Comment (mandresm) [inline docs/cookbook.rst]: The names of the rst files do not correspond with the titles, and therefore, ordering them alphabetically by file name results in no order in the rendered version. I would suggest the following changes:
```
.. include:: recipes/echam_disturbance.rst       # Applying a Temporary...
.. include:: recipes/modify_namelists.rst           # Changing Namelist...
.. include:: recipes/add_model_setup.rst          # Implement a New...
.. include:: recipes/use_own_namelist.rst         # Using Your Own...
```

File: docs/recipes/echam_disturbance.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
+Applying a temporary disturbance to ECHAM
```
Comment (mandresm) [inline docs/recipes/echam_disturbance.rst]: Through out the document we follow the convention of capitalizing the everything that is not an article or a preposition. I am not the biggest fan of it but should we stick to it?

File: docs/recipes/echam_disturbance.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+       30: 
+       30: FATAL ERROR in cuadjtq (1):  lookup table overflow
+       30:  FINISH called from PE: 30
+
+To overcome this problem, you can apply a small change to the air viscosity via
+the namelist parameter ``ensdiff`` in the ``dynctl`` section of the ``ECHAM``
+namelist. As this is a common problem, there is a way to have the run do this
+automatically for some specific years of your simulation.
+
+1. Generate a file to list years you want distrubed.
```
Comment (mandresm) [inline docs/recipes/echam_disturbance.rst]: there is a typo in `disturbed`

File: docs/recipes/echam_disturbance.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+      | | - run_config.yaml
+      | | - distrub_years.dat
+      | |
+      | * outdata
+      | | - echam
+      | ... other files and folders ...
+
+
+2. Add years you want disturbed
+   The file should contain a list of years the distrubance should be applied
```
Comment (mandresm) [inline docs/recipes/echam_disturbance.rst]: typo in `disturbance`

File: docs/recipes/echam_disturbance.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+In this example, we disturb the years 2005, 2007, and 2008 of an experiment
+called ``EXAMPLE`` running on ``ollie``::
+
+  $ cat /work/ollie/pgierz/test_esmtools/EXAMPLE/scripts/disturb_years.dat 
+  2005
+  2007
+  2008
+
+You can also set the distrubance strength in your configuration under
```
Comment (mandresm) [inline docs/recipes/echam_disturbance.rst]: typo in `disturbance`

File: docs/recipes/echam_disturbance.rst [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+``echam.disturbance``. The default is ``1.000001``. Here, we apply a 200%
+distrubance whenever a "distrub_year" is encountered.
```
Comment (mandresm) [inline docs/recipes/echam_disturbance.rst]: typo in `disturbance` and `"disturb_year"`

**Review by mandresm — CHANGES_REQUESTED** [review]

File: docs/recipes/echam_disturbance.rst (hunk 7 above)
Comment (pgierz) [reply docs/recipes/echam_disturbance.rst]: solved by christian

File: docs/recipes/echam_disturbance.rst (hunk 6 above)
Comment (pgierz) [reply docs/recipes/echam_disturbance.rst]: solved by christian

File: docs/recipes/echam_disturbance.rst (hunk 5 above)
Comment (pgierz) [reply docs/recipes/echam_disturbance.rst]: solved by christian

File: docs/recipes/echam_disturbance.rst (hunk 4 above)
Comment (pgierz) [reply docs/recipes/echam_disturbance.rst]: solved by christian

File: docs/cookbook.rst (hunk 2 above)
Comment (pgierz) [reply docs/cookbook.rst]: Reorganized based upon actual title

File: docs/recipes/echam_disturbance.rst (hunk 1 above)
Comment (pgierz) [reply docs/recipes/echam_disturbance.rst]: That'd be nice!

File: docs/recipes/echam_disturbance.rst (hunk 3 above)
Comment (pgierz) [reply docs/recipes/echam_disturbance.rst]: Christian now suggested an even longer title...Maybe we should find a nice system for this?

**Pull request merged**